common/src/main/java/org/apache/drill/exec/metrics/CpuGaugeSet.java

```diff
@@ -0,0 +1,62 @@
+/**

```

Comment:
Javadoc should be changed to comment.

Reply:
Never noticed that the license header is actually a javadoc comment. Will fix this.